modules/auxiliary/scanner/http/http_login.rb

```diff
@@ -171,6 +171,7 @@ def run_host(ip)
         cred_details: cred_collection,
         stop_on_success: datastore['STOP_ON_SUCCESS'],
         bruteforce_speed: datastore['BRUTEFORCE_SPEED'],
+        ssl: datastore['SSL'],
```

Comment:
Does this line belong in `configure_http_login_scanner` ? 🤔 

It looks like only one other scanner honors this flag:

```
modules/auxiliary/scanner/http/jenkins_login.rb
42:      configure_http_login_scanner(
43-        uri: datastore['TARGETURI'],
44-        ssl: datastore['SSL'],
45-        method: datastore['HTTP_METHOD'],
46-        cred_details: cred_collection,
47-        stop_on_success: datastore['STOP_ON_SUCCESS'],
48-        bruteforce_speed: datastore['BRUTEFORCE_SPEED'],
49-        connection_timeout: 10,
50-        http_username: datastore['HttpUsername'],
51-        http_password: datastore['HttpPassword']
52-      )
```

So this same bug is likely to exist in all of the other http based login scanner modules? 🤔